packages/server-wallet/src/__test__/create-channel/fake-funding.test.ts

```diff
@@ -49,7 +49,7 @@ it('Create a fake-funded channel between two wallets ', async () => {
     allocations: [allocation],
     appDefinition: ethers.constants.AddressZero,
     appData: '0x00', // must be even length
-    fundingStrategy: 'Direct',
+    fundingStrategy: 'Unfunded',
```

Comment:
The name of the test file  is 'fake funding'. I think we decided that 'unfunded' should be deprecated terminology?

Reply:
Yeah I figure we could rename it to FakeFunded in a separate PR?

Reply:
I agree that “fake-funded” is better in this case. To me “unfunded” sounds a lot like, “not funded (yet/anymore)” rather than “it’s fine to progress, as we’re never going to fund it”.